account_payment_term_cutoff_day/models/account_payment_term.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountPaymentTerm(models.Model):
+    _inherit = "account.payment.term"
+
+    has_cutoff_day = fields.Boolean(
+        help="If enabled, the due date will be moved to the following month if "
+        "the invoice date exceeds the cutoff day."
+    )
```

Comment:
I now think we should move `has_cutoff_day` to `account.payment.term.line` and delete this file.
```suggestion
```